tldraw/tldraw
# Fix locale changes.

File names in the sidebar did not change when you changed the language. This fixes it.

### Change type


### Test plan

1. Create some files that don't have names set (they show file creation date).
2. Change the language.
3. It should update the filenames accordingly.
